### PR TITLE
Installation and usage Guide. 

### PR DESCRIPTION
**Pull request by ColdForeign**

### CODE REVIEW

File: src/Cropper.Blazor/Client/Models/Snippets.cs [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            if (field == null)
+                return $"Snippet for component '{component}' not found!";
+            return (string)field.GetValue(null);
+        }
+
+        // used for webapi examples
+        public const string Element = @"using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace MudBlazor.Examples.Data.Models
```
Comment (MaxymGorn) [inline src/Cropper.Blazor/Client/Models/Snippets.cs:22]: Please use our namespace

File: src/Cropper.Blazor/Client/Components/SectionContent.razor.cs [hunk 2]
```diff
@@ -0,0 +1,193 @@
+﻿// Copyright (c) MudBlazor 2021
+// MudBlazor licenses this file to you under the MIT license.
```
Comment (MaxymGorn) [inline src/Cropper.Blazor/Client/Components/SectionContent.razor.cs:2]: Remove this comments

File: src/Cropper.Blazor/Client/Components/SectionContent.razor.cs [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+            if (Regex.Match(codeFiles, @"\bServer\b").Success)
+            {
+                var serverCodeFile = "Server.cs" + (char)31 + Snippets.GetCode("Server");
+                codeFiles = codeFiles + (char)31 + serverCodeFile;
+            }
+        }
+
+        var codeFileEncoded = codeFiles.ToCompressedEncodedUrl();
+        // var tryMudBlazorLocation = "https://localhost:5001/";
+        var tryMudBlazorLocation = "https://try.mudblazor.com/";
```
Comment (MaxymGorn) [inline src/Cropper.Blazor/Client/Components/SectionContent.razor.cs:189]: Remove this comment

File: src/Cropper.Blazor/Client/Models/Snippets.cs [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        public override string ToString()
+        {
+            return $""{Sign} - {Name}"";
+        }
+    }
+}";
+
+        public const string Server = @"using System;
+
+namespace MudBlazor.Examples.Data.Models
```
Comment (MaxymGorn) [inline src/Cropper.Blazor/Client/Models/Snippets.cs:45]: Pls use our namespace

File: src/Cropper.Blazor/Client/Components/SectionContent.razor.cs [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+                        read = Regex.Replace(read, $"{HighLight}(?=\\s|\")", $"<mark>$&</mark>");
+                    }
+                }
+
+                builder.AddMarkupContent(0, read);
+            }
+        }
+        catch (Exception)
+        {
+            // todo: log this
```
Comment (MaxymGorn) [inline src/Cropper.Blazor/Client/Components/SectionContent.razor.cs:139]: Pls log this error to console

File: src/Cropper.Blazor/Client/Models/CodeFile.cs [hunk 6]
```diff
@@ -0,0 +1,7 @@
+﻿// Copyright (c) MudBlazor 2021
+// MudBlazor licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Cropper.Blazor.Client.Models;
+
```
[the remark this answers is not in the document]
Comment (ColdForeign) [reply src/Cropper.Blazor/Client/Models/CodeFile.cs:6]: No, it's file scoped namespace

File: src/Cropper.Blazor/Client/Models/Snippets.cs (hunk 4 above)
Comment (ColdForeign) [reply src/Cropper.Blazor/Client/Models/Snippets.cs]: This lines removed.

File: src/Cropper.Blazor/Client/Pages/CropperDemo.razor.cs [hunk 7]
```diff
@@ -21,7 +24,13 @@ public partial class CropperDemo : IDisposable
 
         private CropperComponent? cropperComponent = null!;
         private CropperDataPreview? cropperDataPreview = null!;
-        private Options options = null!;
+        public Options Options = new Options()
```
Comment (MaxymGorn) [inline src/Cropper.Blazor/Client/Pages/CropperDemo.razor.cs:27]: why we need this changes?

File: src/Cropper.Blazor/Client/wwwroot/decode.min.js [hunk 8]
```diff
@@ -1 +1 @@
...
...5P=Q\fQ\bP5P<P=P>P3P8Q\x05P4P0P=P=P>P9P7P=P0Q\x07P8Q\x02P=P5P;Q\fP7Q\x0fQ\x04P>Q\0Q\x03P<P0P"P5P?P5Q\0Q\fP<P5Q\x01Q\x0fQ\x06P0P7P0Q\tP8Q\x02Q\vP\x1bQ\x03Q\x07Q\bP8P5`$(`$9`%\0`$\x02`$\x15`$0`$(`%\x07`$\x05`$*`$(`%\x07`$\x15`$?`$/`$>`$\x15`$0`%\x07`$\x02`$\x05`$(`%\r`$/`$\x15`%\r`$/`$>`$\x17`$>`$\x07`$!`$,`$>`$0`%\x07`$\x15`$?`$8`%\0`$&`$?`$/`$>`$*`$9`$2`%\x07`$8`$?`$\x02`$9`$-`$>`$0`$$`$\x05`$*`$(`%\0`$5`$>`$2`%\x07`$8`%\x07`$5`$>`$\x15`$0`$$`%\x07`$.`%\x07`$0`%\x07`$9`%\v`$(`%\x07`$8`$\x15`$$`%\x07`$,`$9`%\x01`$$`$8`$>`$\x07`$\x1f`$9`%\v`$\x17`$>`$\x1c`$>`$(`%\x07`$.`$?`$(`$\x1f`$\x15`$0`$$`$>`$\x15`$0`$(`$>`$\t`$(`$\x15`%\x07`$/`$9`$>`$\x01`$8`$,`$8`%\x07`$-`$>`$7`$>`$\x06`$*`$\x15`%\x07`$2`$?`$/`%\x07`$6`%\x01`$0`%\x02`$\x07`$8`$\x15`%\x07`$\x18`$\x02`$\x1f`%\x07`$.`%\x07`$0`%\0`$8`$\x15`$$`$>`$.`%\x07`$0`$>`$2`%\x07`$\x15`$0`$\x05`$\'`$?`$\x15`$\x05`$*`$(`$>`$8`$.`$>`$\x1c`$.`%\x01`$\x1d`%\x07`$\x15`$>`$0`$#`$9`%\v`$$`$>`$\x15`$!`$<`%\0`$/`$9`$>`$\x02`$9`%\v`$\x1f`$2`$6`$,`%\r`$&`$2`$?`$/`$>`$\x1c`%\0`$5`$(`$\x1c`$>`$$`$>`$\x15`%\b`$8`%\x07`$\x06`$*`$\x15`$>`$5`$>`$2`%\0`$&`%\x07`$(`%\x07`$*`%\x02`$0`%\0`$*`$>`$(`%\0`$\t`$8`$\x15`%\x07`$9`%\v`$\x17`%\0`$,`%\b`$ `$\x15`$\x06`$*`
```
Comment (MaxymGorn) [inline src/Cropper.Blazor/Client/wwwroot/decode.min.js:1]: pls revert this changes

File: src/Cropper.Blazor/Cropper.Blazor/wwwroot/cropperJsInterop.min.js [hunk 9]
```diff
@@ -0,0 +1 @@
...zoomTo(ratio,{pivotX,pivotY})}noConflict(){return Cropper.noConflict()}setDefaults(options){return Cropper.setDefaults(options)}async getImageUsingStreaming(imageStream){const arrayBuffer=await imageStream.arrayBuffer(),blob=new Blob([arrayBuffer]);return URL.createObjectURL(blob)}revokeObjectUrl(url){URL.revokeObjectURL(url)}initCropper(image,optionsImage,imageObject){if(image==null)throw"Parameter 'image' must be is not null!";const options={};imageObject!=null&&(options.ready=function(event){imageObject.invokeMethodAsync("IsReady",event)},options.cropstart=function(event){imageObject.invokeMethodAsync("CropperIsStarted",event.detail)},options.cropmove=function(event){imageObject.invokeMethodAsync("CropperIsMoved",event.detail)},options.cropend=function(event){imageObject.invokeMethodAsync("CropperIsEnded",event.detail)},options.crop=function(event){imageObject.invokeMethodAsync("CropperIsCroped",event.detail)},options.zoom=function(event){imageObject.invokeMethodAsync("CropperIsZoomed",event.detail)});optionsImage!=null&&Object.entries(optionsImage)?.forEach(([key,value])=>{options[key]=value});this.cropperInstance=new Cropper(image,options)}}window.cropper=new CropperDecorator;
```
Comment (MaxymGorn) [inline src/Cropper.Blazor/Cropper.Blazor/wwwroot/cropperJsInterop.min.js:1]: Remove this file

File: src/Cropper.Blazor/Cropper.Blazor.UnitTests/Cropper.Blazor.UnitTests.csproj [hunk 10]
```diff
@@ -41,4 +41,16 @@
 		<ProjectReference Include="..\Cropper.Blazor\Cropper.Blazor.csproj" />
 	</ItemGroup>
 
+  <!--https://github.com/microsoft/vstest/issues/4014-->
+  <!--Remove section after dotnet SDK 7.0.101 released-->
+  <PropertyGroup>
+    <CollectCoverage>true</CollectCoverage>
```
Comment (MaxymGorn) [inline src/Cropper.Blazor/Cropper.Blazor.UnitTests/Cropper.Blazor.UnitTests.csproj:47]: Okok, pls remove same logic from pipeline step in CI, CD

**Review by MaxymGorn — APPROVED** [review]

**Review by MaxymGorn — APPROVED** [review]

**Review by MaxymGorn — APPROVED** [review]

**Review by MaxymGorn — APPROVED** [review]
